### DISCUSSION TITLE
blog/postgres-tidbit

### DISCUSSION THREAD

**Discussion by giscus** [Austionian/blog.r00ks#32] [Announcements]
# blog/postgres-tidbit

You should never update an app's directly with a SQL statement. (It's an easy way to forget a WHERE clause and update an entire table). But sometimes there's exceptions and sometimes there's neat behavior I've never seen before.

http://localhost:3000/blog/postgres-tidbit

**Comment by Austionian** [comment]
First comment. Woohoo!

**Comment by I-am-tk** [comment]
Second commet!!!!
